packages/fiber/src/core/index.tsx

```diff
@@ -179,6 +179,7 @@ function createRoot<TCanvas extends Element>(canvas: TCanvas): ReconcilerRoot<TC
       const toneMapping = flat ? THREE.NoToneMapping : THREE.ACESFilmicToneMapping
       if (gl.outputEncoding !== outputEncoding) gl.outputEncoding = outputEncoding
       if (gl.toneMapping !== toneMapping) gl.toneMapping = toneMapping
+      if (!gl.physicallyCorrectLights) gl.physicallyCorrectLights = true
```

Comment:
Shouldn't this check for undefined instead? So that it defaults to true only when undefined, otherwise it would always end up true (eg. 

```
gl={{
  physicallyCorrectLights: false
}}
```

Reply:
I agree, it's also more inline with the above statements.

Reply:
I don't understand the difference, gl options are applied afterwards.

Reply:
I don't see the difference, gl options are applied afterward. I'll make it more inline with the other options since this can be a bit error-prone, especially if coupled with a flag.